politeiad/plugins/comments/comments.go

```diff
@@ -463,9 +463,19 @@ type CountReply struct {
 	Count uint32 `json:"count"`
 }
 
-// Votes retrieves the comment votes that meet the provided filtering criteria.
+const (
+	// VotesPageSize is the maximum number of comment votes
+	// that can be requests at any one time.
+	VotesPageSize uint32 = 100
+)
```

Comment:
This is defined twice, here and in `politeiawww/api/comments/v1/v1.go`, it's needed in both places as we 
need to return in the comments policy reply and we also need it in politeiad to use it when returning  a paginated reply. 

Is there a way to improve that ? maybe we could add it as plugin setting ?

Reply:
I agree, converting these page sizes into plugin settings is probably the right way to do it.